prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <a href="/Users/mikeh/repos/prep-0621-code-solutions/prep-html-links/index.html">Click here to go to the home page</a>
```

Comment:
This `href` is an "absolute" path which requires all of the same files and folders of your personal computer, but we want to use a "relative" path.
This one should be `href="../index.html"` only.